src/components/container/ConnectDevicesMenu/ConnectDevicesMenu.tsx

```
@@ -59,7 +73,19 @@ export default function (props: ConnectDevicesMenuProps) {
+                if (deviceInfo?.platform == "iOS") {
+                    MyDataHelps.queryDeviceData({
+                        namespace: "AppleHealth",
+                        observedAfter: formatISO(add(new Date(), { days: -3 })),
+                        limit: 1
+                    }).then((result) => {
+                        setHasRecentAppleHealthData(result.deviceDataPoints.length > 0);
+                        setLoading(false);
+                    });
+                } {
+                    setLoading(false);
+                }
```

Biome (1.9.4)

[error] This block statement doesn't serve any purpose and can be safely removed.

Standalone block statements without any block-level declarations are redundant in JavaScript and can be removed to simplify the code.
Safe fix: Remove redundant block.


(lint/complexity/noUselessLoneBlockStatements)